web/src/main/webapp/js/map.js

```diff
@@ -164,12 +164,12 @@ function initMap(bounds, setStartCoord, setIntermediateCoord, setEndCoord, selec
         },
         contextmenu: true,
         contextmenuItems: [{
-                text: 'Route ',
```

Comment:
here is a space after 'Route' ... maybe we should use a parametrized translation. And did you add those keys to they spreadsheet? If not, I can do it.